homework/index.js

```diff
@@ -0,0 +1,64 @@
+function getPromise() {
+  const ret = fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100')
+    .then(response => response.json());
+  return ret;
+}
+
+
+function populate(Prom) {
+  const selectContent = document.getElementById('liste').value;
+  Prom.then((result) => {
+    result.forEach((x) => {
+      if (x.name === selectContent) {
+        document.getElementById('show').innerHTML = `
+         <b>Repository :</b>   <a href="">${x.name} </a><br>
+          <b>Descripion :</b>   ${x.description} <br>
+          <b>Forks :</b>   ${x.forks} <br>
+          <b>Updated :</b>   ${new Date(x.updated_at).toLocaleString('en-US')} <br>`;
+      }
+    });
+  });
+}
+
+
+function loadPage(Prom) {
```

Comment:
Idem as above: use camelCase for parameters. Also, maybe `populateSelectList` would be a better name to tell what this function's doing?